packages/server/src/api/routes/oauth2.ts

```diff
@@ -28,7 +28,7 @@ function oAuth2ConfigValidator() {
 function oAuth2ConfigValidationValidator() {
   return middleware.joiValidator.body(
     Joi.object({
-      id: Joi.string().required(),
+      id: Joi.string(),
```

Comment:
Why is a separate `id` needed when the config has a `_id`? Is there multiple OAuth configs within the single oauth document?

Reply:
This might validate existing configs or new configs